### PR TITLE
Add Github Shellcheck workflow

### PR DESCRIPTION
**Pull request by pdostal**
There are many bash scripts in this repository and Shellcheck would be useful.

My plan is to enable this workflow but exclude most of the directories so people can individually choose to "opt-in".

### CODE REVIEW

File: data/publiccloud/x
Comment (grisu48) [inline data/publiccloud/x:1]: What is this empty file doing here?

Comment (pdostal) [reply data/publiccloud/x:1]: Sorry, that was just for debugging.

**Review by ricardobranco777 — APPROVED** [review]
LGTM

**Review by grisu48 — APPROVED** [review]

**Review by okurz — COMMENTED** [review]
+1

**Pull request merged**